projects/igniteui-angular/src/lib/drop-down/drop-down.component.ts

```diff
@@ -242,7 +255,8 @@ export class IgxDropDownComponent extends IgxDropDownBaseDirective implements ID
      * ```
      */
     public open(overlaySettings?: OverlaySettings) {
-        this.toggleDirective.open(overlaySettings);
+        const settings = overlaySettings || { ...defaultOverlaySettings };
```

Comment:
spreading the `defaultOverlaySettings` is correct. However, this will not create a clone of the positioning strategy. If at later point someone changes its position settings this will break the default ones. To fix this either make `defaultOverlaySettings` a getter which returns always a new instance of `OverlaySettings` or method doing the same.